app/Constants/ControllerRating.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace App\Constants;
+
+final class ControllerRating
+{
+    private const RATINGS = [
+        'OBS' => 'Observer',
+        'S1' => 'Student 1',
+        'S2' => 'Student 2',
+        'S3' => 'Senior Student',
+        'C1' => 'Controller 1',
+        'C3' => 'Controller 3'
+    ];
+
+    private const NUMERIC_RATINGS = [
+        0 => 'OBS',
+        1 => 'S1',
+        2 => 'S2',
+        3 => 'S3',
+        4 => 'C1',
+        5 => 'C3'
+    ];
+
+    private const SUFFIX_REQUIREMENTS = [
+        'GND' => 1,
+        'TWR' => 2,
+        'APP' => 3,
+        'CTR' => 4
```

Comment:
Need to account for FSS, DEL and likely others too. Can these be defined somewhere central so that it isn’t a repeated list as the positions model? Not sure.

Reply:
Doh! Will amend. They are storing two different units of information, but I do see your point. Can't really think of anything either.

Reply:
Could we define a rating requirement as an attribute on a position?

Reply:
We could, but I envisioned us only doing that for any specific cases e.g. endorsements, and it was better to have a global handler for it. I think when nothing will change, this represents the most optimal solution at the moment.